Automattic/wp-calypso
# Fix lint errors in JetpackSSOForrm

Update to es6 class.
Fix usage of this.translate.

Lint errors must be fixed before jetpack-connect can be moved.

To test:

- You must have a connected Jetpack site with SSO module (AKA WordPress.com log in Security settings) enabled.
- Create a new user on the site.
- You must have (or create) a different WordPress.com user to associate with the new site user via SSO.
- All instructions refer to the new user, not the main user who connected the site to WordPress.com

1. Ensure you are logged out.
- Go to /wp-login.php of your Jetpack site.
- Click the “Log in with WordPress.com” button.
- Follow the SSO flow.
- Ensure there are no regressions. Watch the console (preserve log) for any errors/warnings.

Should be merged before #12991.
